pkg/modulewriter/modulewriter.go

```diff
@@ -36,12 +36,12 @@ import (
 
 // strings that get re-used throughout this package and others
 const (
-	HiddenGhpcDirName          = ".ghpc"
-	ArtifactsDirName           = "artifacts"
-	ExpandedBlueprintName      = "expanded_blueprint.yaml"
-	prevDeploymentGroupDirName = "previous_deployment_groups"
-	gitignoreTemplate          = "deployment.gitignore.tmpl"
-	artifactsWarningFilename   = "DO_NOT_MODIFY_THIS_DIRECTORY"
+	HiddenGhpcDirName        = ".ghpc"
+	ArtifactsDirName         = "artifacts"
+	ExpandedBlueprintName    = "expanded_blueprint.yaml"
+	prevGroupDirName         = "previous_deployment_groups"
```

Comment:
nit: previous_deployment_groups  -> previous_groups?

Reply:
`previous_deployment_groups` is a name of folder, I wouldn't not change the layout to avoid making this a "breaking change"